hermes-console-vue/src/views/favorite/topics/topic-listing/FavoriteTopicsListing.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+  import { computed } from 'vue';
+  import { groupName } from '@/utils/topic-utils/topic-utils';
+  import { useFavorites } from '@/store/favorites/useFavorites';
+  import { useI18n } from 'vue-i18n';
+  import { useRouter } from 'vue-router';
+
+  const router = useRouter();
+  const { t } = useI18n();
+
+  const favorites = useFavorites();
+
+  const props = defineProps<{
+    topics?: string[];
+    filter?: string;
+  }>();
+
+  const filteredTopics = computed(() => {
+    return (props.topics ?? []).filter(
+      (topic) => !props.filter || topic.includes(props.filter),
+    );
+  });
+
+  function onTopicClick(topicName: string) {
+    router.push({
+      path: `/ui/groups/${groupName(topicName)}/topics/${topicName}`,
+    });
+  }
+</script>
+
+<template>
+  <v-card class="mb-2">
+    <v-table density="comfortable" hover>
+      <thead>
+        <tr>
+          <th>{{ t('groups.groupTopicsListing.index') }}</th>
+          <th>{{ t('groups.groupTopicsListing.name') }}</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody v-if="filteredTopics.length > 0">
+        <tr
+          v-for="(topic, index) in filteredTopics"
+          :key="topic"
+          class="topics-table__row"
+        >
+          <td class="text-medium-emphasis" @click="onTopicClick(topic)">
```

Comment:
question: is there a way to make the whole row trigger `onTopicClick` except the area of `favourites icon` which should trigger `favorites. removeTopic` (so that we don't have to use `onTopicClick` for each cell)?

Reply:
I tried to do it for the whole row (I mean redirect on click), but I failed to overwrite this action on the button